source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,384 @@ Sort Results
+
+.. _sorts-crud-text-search:
+
+Text Search
+-----------
+
+You can specify the order of the results of a 
+:manual:`text search </text-search/>` by how closely they match your
+search string. Each of your search results has a
+:manual:`text score </reference/operator/aggregation/meta/#exp._S_meta>` which is a
+numerical value that indicates how closely that result matches your search. 
+Use the ``Sorts.metaTextScore()`` static factory method to build your sort
+criteria to sort by the text score.
+
+.. warning:: Make Sure to Create a Text Index
+
+   You need a :manual:`text index </core/index-text/>` on your collection to
+   perform a text search. See the server manual documentation for more
+   information on how to
+   :manual:`create a text index </core/index-text/#create-text-index>`.
+
+In the following code example, we show how you can use the
+``Sorts.metaTextScore()`` method to sort the results of a text
+search on the :ref:`sort_example collection <sorts-crud-sort-example>`.
+The code example uses the :doc:`Filters </fundamentals/builders/filters>`,
+:doc:`Indexes </fundamentals/builders/indexes>`, and
+:doc:`Projections </fundamentals/builders/projections>` builders.
+The code example performs the following actions:
+
+#. Creates a text index for your
+   :ref:`sort_example collection <sorts-crud-sort-example>`
+   on the ``food`` field. If the index specified in your call to the
+   ``createIndex()`` method already exists, calling it will not create a new
+   index.
+#. Runs your text search for the phrase "maple donut".
+#. Projects text scores into your query results as the
+   ``score`` field. This projection is optional if your MongoDB instance is
+   running MongoDB 4.4 or later. 
+#. Sorts your results by text score (best match first).
+
+.. code-block:: java
+
+   import com.mongodb.client.model.Sorts;
+   import com.mongodb.client.model.Projections;
+   import com.mongodb.client.model.Filters;
+   import com.mongodb.client.model.Indexes;
+
+   // <MongoCollection setup code here>
+   
+   collection.createIndex(Indexes.text("food"));
+   Bson metaTextScoreSort = Sorts.metaTextScore("score");
+   Bson metaTextScoreProj = Projections.metaTextScore("score");
+   String searchTerm = "maple donut";
+   Bson searchQuery = Filters.text(searchTerm);
+   collection.find(searchQuery)
+            .projection(metaTextScoreProj)
+            .sort(metaTextScoreSort)
+            .into(results);
+   for (Document result : results) {
+         System.out.println(result.toJson());
+   }
+
+The output of the code example above should look something like this: 
+
+.. code-block:: json
+
+   {"_id": 6, "letter": "c", "food": "maple donut", "score": 1.5}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee", "score": 0.75}
+   {"_id": 3, "letter": "a", "food": "maple syrup", "score": 0.75}
+
+.. note:: ``$meta`` Behavior in MongoDB 4.4 or Later
+
+  If you are using MongoDB 4.4 or later, projecting
```

Comment:
This should be indented 3 spaces to line up with the "n" in "note" similar to what you have for the warning admonition you used earlier.

Reply:
Thanks for the note. I'm surprised this did not raise any errors (that I am aware of anyway). Going forward, I will make these checks manually. Fix: https://github.com/mongodb/docs-java/pull/66/commits/fc5d2ad9552aa5f7de81850a8090b66337683917